scene/3d/camera.cpp

```diff
@@ -523,6 +525,7 @@ void Camera::_bind_methods() {
 	ADD_PROPERTY(PropertyInfo(Variant::VECTOR2, "frustum_offset"), "set_frustum_offset", "get_frustum_offset");
 	ADD_PROPERTY(PropertyInfo(Variant::REAL, "near", PROPERTY_HINT_EXP_RANGE, "0.01,8192,0.01,or_greater"), "set_znear", "get_znear");
 	ADD_PROPERTY(PropertyInfo(Variant::REAL, "far", PROPERTY_HINT_EXP_RANGE, "0.1,8192,0.1,or_greater"), "set_zfar", "get_zfar");
+	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "affects_lod"), "set_affects_lod", "get_affects_lod");
```

Comment:
```suggestion
	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "affect_lod"), "set_affect_lod", "get_affect_lod");
```
I believe this should be active as in "it does affect LOD", otherwise it sounds like "check if it does affect LOD"

Reply:
Affects seems more natural to my ears (as native English speaker)? But I'd be happy to change it, there doesn't seem a lot in it.

It probably comes down to how you think of the phrase being used, it is quite subtle:
* correct - "Does the camera affect LOD?"
* incorrect "Does the camera affects LOD?"
* correct - "Camera affects LOD."
* incorrect - "Camera affect LOD."

Reply:
But here it's "make the camera affect LOD" IMO, it's either way but to me settings read as imperative 🙂

Reply:
Have changed. :+1: